README.md

```diff
@@ -109,24 +103,83 @@ This will ensure that application cache invalidates whenever actual file content
 
 ### Output example
 
-    CACHE MANIFEST
-
-    CACHE:
-    js/app.js
-    css/style
-    css/style.css
-    js/zepto.min.js
-    js/script.js
-    some_files/index.html
-    some_files/about.html
+```
+  CACHE MANIFEST
 
-    NETWORK:
-    *
+  CACHE:
+  js/app.js
+  css/style
+  css/style.css
+  js/zepto.min.js
+  js/script.js
+  some_files/index.html
+  some_files/about.html
 
-    # hash: 76f0ef591f999871e1dbdf6d5064d1276d80846feeef6b556f74ad87b44ca16a
+  NETWORK:
+  *
 
+  # hash: 76f0ef591f999871e1dbdf6d5064d1276d80846feeef6b556f74ad87b44ca16a
+```
 
 You do need to be fully aware of standard browser caching.
 If the files in **CACHE** are in the network cache, they won't actually update,
 since the network cache will spit back the same file to the application cache.
 Therefore, it's recommended to add a hash to the filenames's, akin to rails or yeoman. See [here](http://www.stevesouders.com/blog/2008/08/23/revving-filenames-dont-use-querystring/) why query strings are not recommended.
+
+### Getting correct paths
+
+Sometimes you might want to alter the way paths are passed to the plugin. The correct way will be to provide options to `gulp.src` so that it generates correct paths.
+
+Say, you have a single folder named `public`, which is the top-level directory that's served to the browser. In the same directory, you have the `css`, `js` and `asset` files under different directories, along with the `html` files.
+
+```
+public/
+├── assets
+│   ├── cover.png
+│   └── logo.png
+├── css
+│   └── style.css
+├── js
+│   └── app.js
+└── index.html
+```
+
+Say you wrote your `gulp.src`  (you could use file globbing, but for simplicity's sake, will use separate paths) as following,
+
+```
+gulp.src([
+  'public/assets/**',
+  'public/css/**',
+  'public/js/**'
+])
+```
+
+It'll generate the following paths,
+
+```
+cover.png
+logo.png
+style.css
+app.js
+```
+
+Which is totally unusable. Ideally, what you would want is the following,
```

Comment:
Rather than warning users of this plugin, I prefer to fix the default behavior without `base` specified.